### PR TITLE
Fix load() so it resolves scripts recursively in workflows

### LINKED ISSUE #52: load() does not properly load script URIs for workflow steps
See https://github.com/girder/girder_worker/issues/47#issuecomment-230882875.

### PR DESCRIPTION
**Pull request by jeffbaumes**
Fixes #52.

### CODE REVIEW

**Comment by zachmullen** [comment]
LGTM

File: analysis/xdata/flu_workflow.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "outputs": [
+          {
+            "type": "table",
+            "name": "data",
+            "format": "csv"
+          }
+        ],
+        "mode": "python",
+        "name": "Flu trends",
+        "script_uri": "file://flu.py"
```
Comment (jcfr) [inline analysis/xdata/flu_workflow.json]: To avoid ambiguity, would it make sense to have `"file://./flu.py"` ?

Comment (jeffbaumes) [inline analysis/xdata/flu_workflow.json]: Yes, are you suggesting we add this for clarity here or suggesting that URIs like `file://flu.py` should be made to not function?

Comment (jcfr) [inline analysis/xdata/flu_workflow.json]: > we add this for clarity here

:+1:

**Pull request merged**